ingestion/src/metadata/utils/helpers.py

```diff
@@ -178,3 +191,60 @@ def datetime_to_ts(date: datetime) -> int:
     Convert a given date to a timestamp as an Int
     """
     return int(date.timestamp())
+
+
+def create_lineage(from_table, to_table, query_info):
+    try:
+        from_fqdn = f"{config.service_name}.{from_table}"
+        from_entity = metadata.get_by_name(entity=Table, fqdn=from_fqdn)
+        to_fqdn = f"{config.service_name}.{to_table}"
+        to_entity = metadata.get_by_name(entity=Table, fqdn=to_fqdn)
+        if not from_entity or not to_entity:
+            return None
+        from_entity_id = (
+            from_entity.id.__root__
+            if from_entity and hasattr(from_entity.id, "__root__")
```

Comment:
Why are we checking if `from_entity` and `to_entity` ids have `__root__`? the `get_by_name` call should return a `Table` object. Here, you could annotate the type as:

```python
from_entity: Table = metadata.get_by_name(entity=Table, fqdn=from_fqdn)
```

And then directly call `from_entity.id.__root__`